plugin/src/main/scala-2.12.14/ch/epfl/scala/profilers/tools/SettingsOps.scala

```diff
@@ -0,0 +1,8 @@
+package ch.epfl.scala.profilers.tools
+
+import scala.tools.nsc.Global
+
+object SettingsOps {
+  def areStatisticsEnabled(g: Global): Boolean =
+    g.settings.areStatisticsEnabled
```

Comment:
https://github.com/scala/scala/commit/518e6e076b0a75c4977a876b8ff3d7869f29dcf7

Reply:
I think you can avoid the need for version-dependent source files here using a clever local implicit extension. Something like:
```scala
  implicit class SettingsOps(private val settings: MutableSettings) extends AnyVal {
    @inline final def areStatisticsEnabled = StatisticsStatics.areSomeColdStatsEnabled && settings.YstatisticsEnabled
  }
```
(taken from the commit you mention, actually.

Reply:
Looks like `StatisticsStatics.areSomeColdStatsEnabled` is not available for some Scala versions, and we still need a version-dependend source files for `areStatisticsEnabled`.